packages/graphql/src/types/index.ts

```diff
@@ -449,6 +449,7 @@ export type Neo4jFeaturesSettings = {
      **/
     excludeDeprecatedFields?: {
         implicitEqualFilters?: boolean;
+        implicitSET?: boolean;
```

Comment:
Think the casing is weird here!

```suggestion
        implicitSet?: boolean;
```